exporter/otlp/lib/opentelemetry/exporter/otlp/version.rb

```diff
@@ -8,7 +8,7 @@ module OpenTelemetry
   module Exporter
     module OTLP
       ## Current OpenTelemetry OTLP exporter version
-      VERSION = '0.6.0'
+      VERSION = '0.6.1'
```

Comment:
🤦 I think this may not have been necessary - the release process would have done this. Hopefully, I haven't messed that up.

Reply:
This should be fine. The release process uses release tags to determine the previous/next release, and shouldn't care if the version constant was updated prematurely.